README.md

```diff
@@ -1,4 +1,69 @@
 # potku-console
-Minimalistic server auto start and configuration platform designed to enable running servers only when needed without affecting user experience drastically.
 
-Very much a WIP project, we are currently working on basic documentation and setting up a proper repository.
+"Minimalistic server auto start and configuration platform."
+
+
+
+## Table of content
+* [Status of the project](#status-of-the-project)
+* [What is potku-console](#what-is-potku-console)
+  * [Pitch for the potku-console](#pitch-for-the-potku-console)
+  * [Main (upcoming) features](#main-(upcoming)-features)
+* [Installation instructions](#installation-instructions)
+* [Technologies and architecture](#technologies-and-architecture)
+* [Getting involved and contributions](#getting-involved-and-contributions)
+
+
+
+## Status of the project
+
+This is currently very much a WIP-project. Core team is currently working on basic architecture and documentation of the project.
+
+
+
+## What is potku-console
+
+Potku-console is a service control platform that is user for starting services automatically on demand. Potku-console consists of different kind of modules like: core, frontend, service modules and cloud modules that together form the potku-console. This repository concentrates especially on core and frontend parts, because they form the spine of the potku-console, but also most important first party service and cloud modules are developed in this repository.
+
+
+
+### Pitch for the potku-console
+
+You would like to own a (insert game name here) server where you and your group of Discord friend could play together. You would like to have server with multiple cores and decent amount of RAM from a cloud provider, because the server software is resource hungry and your 10 year old WWW-server computer isn't quite powerful enough. You seek for a good hosting solution, but can't find anything that would fulfill your needs due to tight budget. You ponder that you could save a ton of money in the cloud expenses, if you turn off the server machine when there are no active players on the server. At first idea felt good, but after a minute you figure out that manually controlling the server's power would be impractical, since some of your friends live in different time regions and they like to even after you gone to sleep. If there just would be an easy way to automate this power cycling process...
+
+Yes there is a solution, it is called potku-console. Potku-console monitors automatically player counts on the server and shuts it down after last player has left the game. After the server has been shutdown potku-console sets guard for the servers address that waits for the new players and automatically starts up the server when first player arrives. This arrangement saves ton of money and natural resources.  
```

Comment:
...Potku-console AUTOMATICALLY MONITORS player counts...
...after THE last player...
...potku-console sets A guard...
...for the server's address... (missing apostrophe)
...waits for new players... (no THE with plural form)
...when THE first player...
...saves A ton of money... OR ...saves tonS of money...

Reply:
no THE with plural form is not grammatically correct advice btw, my bad. The "the" doesn't belong there, but plurality has nothing to do with it.